<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labeler_widget</class>
 <widget class="QWidget" name="labeler_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Labeler</string>
  </property>
  <widget class="QLabel" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>421</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_im_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="generate_xlsx_checkbox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Also generate .xlsx file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_im_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_csv_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate csv</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>521</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="show_next_checkbox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatically show next image when labeled</string>
   </property>
  </widget>
  <widget class="QLabel" name="csv_note">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>(csv will be also generated automatically after closing the app)</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_name_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
